t/tars/discovery/tars.t

```diff
@@ -41,8 +41,8 @@ discovery:
       database: db_tars
       user: root
       password: tars2022
-    full_fetch_interval: 3
-    incremental_fetch_interval: 1
+    full_fetch_interval: 90
```

Comment:
Please see [link](https://github.com/apache/apisix/pull/6599#discussion_r827587517)
   >     "full_fetch_interval: 3"
   >     "incremental_fetch_interval: 1"
   I think we should keep it as it is

Reply:
The APISIX will fail before modify the schema, maybe we should figure out other way to reduce the interval.

Reply:
```   
 -- fill the default value by the schema
    schema.validate(default_conf)
```
We didn't handle the result of schema validation before, that's why your solution work, we need to throw error now

Reply:
maybe we should find another way
@spacewander @monkeyDluffy6017

Reply:
> The APISIX will fail before modify the schema, maybe we should figure out another way to reduce the interval.

@monkeyDluffy6017 
Interesting. The modification is done in the init_by_lua phase. It is strange that the schema validation happens before it. Could you dig deeper into it?

Reply:
Sure, i will check this later

Reply:
The modification should be put before apisix = require("apisix")，the validation is called in that.